springwolf-add-ons/springwolf-generic-binding/src/main/java/io/github/springwolf/addons/generic_binding/annotation/processor/PropertiesUtil.java

```diff
@@ -39,17 +43,33 @@ private static Map<String, Object> convertPropertiesToNestedMap(Properties prope
             Map<String, Object> mapNode = bindingData;
             while (path.size() > 1) {
                 String pathElement = path.get(0);
-                if (!mapNode.containsKey(pathElement)) {
-                    mapNode.put(pathElement, new HashMap<>());
-                }
-
+                mapNode.computeIfAbsent(pathElement, k -> new HashMap<>());
```

Comment:
Nice refactor. Is the return value is identical to the get call in the next line? -> combine?

Reply:
It's exactly the same.